### PR TITLE
Device

### PR DESCRIPTION
**Pull request by Errytagedesign**
##In this branch we did:

- Removed the current_user method in the ApplicationController.
- Installed and set up Devise.
- Register a new user.
- Log in using their email and password.
- Store hashed passwords in the database for security.
- Request email confirmation for added security.
- Reset their password if needed.
- Modified the views to align with the wireframe styling, ensuring a consistent and cohesive user interface.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (BenMukebo) [inline app/models/user.rb]: - [ ] Great job so far team, however, it's required for the application to ask for confirmation by email. Please let's address this. 💯 👌🏻 
- You can check the `:confirmable` and enable it in the user model.

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Errytagedesign, @balatstar,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Gitflow was correctly used.  ✔️   
- The current_user method in ApplicationController has been removed.  ✔️ 
- Devise has been installed and setup.  ✔️ 
- A user can log in with a combination of email and password.  ✔️ 
- A user can reset the password in the application.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (Errytagedesign) [reply app/models/user.rb]: I later realized this, i added the :confirmable and after signinup i logged in but i wasn't redirected, to home any tip please?

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @Errytagedesign 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ the confirmable method has been added
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**